### PR TITLE
5 write endpoints

### LINKED ISSUE #5: Create Endpoints
### Possible Endpoints:

| Data | Method |
| --- | --- |
| Product | GET (by ID) |
| | POST |
| | PATCH |
| | DELETE |
| User | GET (by ID) |
| | POST |
| | PATCH |
| | DELETE |
| User Favorite | POST |
| | DELETE |
| Products | GET (by Prefecture) |
| | GET (by Campaign End) |
| | GET (by User, i.e., in User Favs.) |
| | GET (top 10, ranked by Fav. count) |
| User Favorites | GET (by User) |

### PR DESCRIPTION
**Pull request by nikobatzi1990**
Resolves #5 

### REQUIRED:
- [x] Write the issue number associated with this pull request after "Resolves #"
- [x] Add the correct label for this pull request (you can just use the same label as the associated issue)
- [x] Be sure to share a link to this pull request in Slack/Discord and ask for a review! <3

### CODE REVIEW

File: config/settings.py [hunk 1]
```diff
@@ -32,7 +31,7 @@
 SECRET_KEY = env('SECRET_KEY')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = env('DEBUG')
+DEBUG = False
```
Comment (kappanjoe) [inline config/settings.py:34]: This should remain tied to the environment variable - could you revert this change or maybe include a default if the environment variable doesn't exist?

**Review by kappanjoe — CHANGES_REQUESTED** [review]
Just one small change - rest looks great!

File: config/settings.py (hunk 1 above)
Comment (nikobatzi1990) [reply config/settings.py]: Just reverted the change in my latest commit!

**Review by kappanjoe — APPROVED** [review]
Thank you so much! Everything seems great!
